datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxRetryService.java

```diff
@@ -69,4 +78,8 @@ private boolean isCloudbreakRetryNecessary(String retriedEvent) {
         );
     }
 
+    private List<Class<? extends FlowConfiguration<?>>> getFullRetryFlows() {
+        return Arrays.asList(DatalakeRestoreFlowConfig.class,
```

Comment:
I think this should be a final global variable it seems kind of weird to have a method for this and would make it clearer that we are defining an explicit list of flows which follow this logic.

Reply:
I agree and this could be a set as well.